models/system_request_handler.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+"""
+module to handle all incoming requests for server side information
+"""
+
+from models.exceptions import *
+from models.return_data_object import ReturnDataTransferObject
+
+class system_request_handler:
```

Comment:
Class name should follow [PascalCase](https://www.theserverside.com/definition/Pascal-case) convention.

Reply:
@Matthew-brinkmann On slack, you shared with me your thought about making this a class or a python file with methods.
I think a class is fine, but i don't feel you need any instance of it. So you can make this one a static class, implementing only static methods.